### PR TITLE
Add cozy-stack missing doctypes

### PR DESCRIPTION
**Pull request by maximemenager**

### CODE REVIEW

File: docs/io.cozy.jobs.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  - `push`: This worker is responsible of mobile notifications
+  - `share-*`: `share-track`, `share-replicate` & `share-upload` manage all the
+    share-related data between the instances.
+  - `unzip`: Unzip files
+  - `updates`: Handle app/konnectors updates
+- `message` {object}: Contains all the specific-worker data (slug, script name,
+  ...)
+- `manual_execution` {bool}: Tells if the job had beed manually excuted
+- `event` {object}: Holds the informations for realtime
+- `options` {object}: Custom overrided options for the job (`max_exec_count`, `max_exec_time`, `timeout`)
```
Comment (drazik) [inline docs/io.cozy.jobs.md]: Custom overriden

File: docs/io.cozy.triggers.state.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+## `io.cozy.triggers.state`
+
+The `io.cozy.triggers.state` doctype gather the data of a trigger from its jobs
+
+- `status` {string}: Global status of the trigger (`queued`, `running`, `done`, `errored`)
+- `last_success` {timestamp}: Date of the last job in success
+- `last_successful_job_id` {string}: ID of the last job in success
+- `last_execution` {timestamp}: Date of the last job executed
+- `last_executed_job_id` {string}: ID of the last job in executed
```
Comment (drazik) [inline docs/io.cozy.triggers.state.md]: The last executed job

File: docs/README.md [hunk 3]
```diff
@@ -3,18 +3,30 @@
...
+- [Apps](io.cozy.apps.md): Apps
 - [Autocategorization](cc.cozycloud.autocategorization.md): Auto categorization remote doctype
 - [Bank](io.cozy.bank.md): banking related data
 - [Bills](io.cozy.bills.md): bills
 - [Contacts](io.cozy.contacts.md): instance owner contacts
+- [Exports](io.cozy.exports.md): Instances exports
 - [Files](io.cozy.files.md): files documents
-- [Konnectors](io.cozy.konnectors): Connectors
+- [Jobs](io.cozy.jobs.md): Instance jobs
+- [Konnectors](io.cozy.konnectors.md): Konnectors
```
Comment (edas) [inline docs/README.md]: I would have leaved "Connectors" as is as a description.

File: docs/io.cozy.account_types.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+- It can contain secret configuration parameters needed by the webapp/konnectors
+  (API Tokens, API URLS, ...). Every time a konnector job is executed, the stack
+  can give some of these secret parameters to the konnector
+
+- It can contain an OAuth configuration for a service. It makes possible for the
+  stack to talk to external services without allowing the konnector to directly
+  access the outside world
+
+
+- `grant_mode` {string}: Grant mode, part of the OAuth standard
```
Comment (edas) [inline docs/io.cozy.account_types.md]: OAuth talks about grant type, is this the same ?

File: docs/io.cozy.sessions.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
+- [Cozy sessions doctype](#cozy-sessions-doctype)
+  - [`io.cozy.sessions`](#iocozysessions)
+
+# Cozy sessions doctype
+
+## `io.cozy.sessions`
+
+The `io.cozy.sessions` doctype holds the web sessions informations.
+
+- `instance` {object}: The instance of the sessions
```
Comment (edas) [inline docs/io.cozy.sessions.md]: Do you have an example ?

File: docs/io.cozy.settings.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
+- [Cozy settings doctype](#cozy-settings-doctype)
+  - [`io.cozy.settings`](#iocozysettings)
+
+# Cozy settings doctype
+
+## `io.cozy.settings`
+
+The `io.cozy.settings` doctype contains all the stack instance-related settings.
+
+- `tz` {string}: Timezone of the instance
```
Comment (edas) [inline docs/io.cozy.settings.md]: as `+01:00` ? as "Europe/Paris" ? Something else ?

File: docs/io.cozy.account_types.md (hunk 4 above)
Comment (maximemenager) [reply docs/io.cozy.account_types.md]: Yes

**Comment by nono** [comment]
[(𝐌) [DOCTYPE] Lister l'ensemble des docType géré par la stack](https://trello.com/c/Qw1HI3Ba/1389-%F0%9D%90%8C-doctype-lister-lensemble-des-doctype-g%C3%A9r%C3%A9-par-la-stack)

File: docs/README.md [hunk 7]
```diff
@@ -3,18 +3,33 @@
...
+- [Exports](io.cozy.exports.md): Instances exports
+- [Jobs](io.cozy.jobs.md): Instance jobs
+- [Notifications](io.cozy.notifications.md): notifications made by the apps
+- [OAuth Clients](io.cozy.oauth.clients.md): OAuth clients informations of the stack
+- [OAuth Access Codes](io.cozy.oauth.access_codes.md): Access codes specific to OAuth flow
 - [Triggers](io.cozy.triggers.md): Job triggers
+- [Triggers state](io.cozy.triggers.state.md): Job triggers states
+- [Remote requests](io.cozy.remote.requests.md): logs of requests via the remote doctypes
+- [Sessions](io.cozy.sessions.md): cozy-stack sessions
+- [Sessions Logins](io.cozy.sessions.logins.md): sessions logins entry
```
Comment (nono) [inline docs/README.md]: I'm a bit surprized by which doctypes are in cozy doctypes and which doctypes are in technical doctypes. For example, I'd have put `io.cozy.shared` in technical, as it is only an implementation detail viewed by the stack. But notifications or sessions logins are similar to settings, and I would have put them in Cozy doctypes.

File: docs/io.cozy.account_types.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+
+The `io.cozy.account_types` can be used for two main purposes:
+
+- It can contain secret configuration parameters needed by the webapp/konnectors
+  (API Tokens, API URLS, ...). Every time a konnector job is executed, the stack
+  can give some of these secret parameters to the konnector
+
+- It can contain an OAuth configuration for a service. It makes possible for the
+  stack to talk to external services without allowing the konnector to directly
+  access the outside world
```
Comment (nono) [inline docs/io.cozy.account_types.md]: No, the konnector can still access the outside world, and the OAuth provider in particular. It is just a way to simplify the development of konnectors for OAuth providers, and it avoid race conditions when the same account can be used by several konnectors (I think of the example of google).

File: docs/io.cozy.sessions.md [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+  - [`io.cozy.sessions`](#iocozysessions)
+  - [Example](#example)
+
+# Cozy sessions doctype
+
+## `io.cozy.sessions`
+
+The `io.cozy.sessions` doctype holds the web sessions informations.
+
+- `instance` {object}: The instance of the sessions
```
Comment (nono) [inline docs/io.cozy.sessions.md]: I think you should remove the `instance` field: it is used purely inside the stack (in memory), but not persisted to CouchDB or accessible by the clients.

File: docs/io.cozy.shared.md [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+The `io.cozy.shared` doctype is related to the sharing part of the stack. It
+tracks the sharings and its references, its infos & revisions across the stack.
+
+- `revisions` {object}: Tree of revisions of a sharing
+- `infos` {map[string]object}: Infos of the sharing
+  - `rule` {int}: Index of the rule inside the sharing rules
+  - `removed` {bool}: Either the document has been deleted, the file is trashed,
+    or the document does no longer match the sharing rule
+  - `binary` {bool}: Only true if the sharing is a file and has not been
+    trashed
```
Comment (nono) [inline docs/io.cozy.shared.md]: Maybe we can add a link to https://docs.cozy.io/en/cozy-stack/sharing-design/#iocozyshared

File: docs/io.cozy.triggers.state.md [hunk 11]
```diff
@@ -0,0 +1,19 @@
+- [Cozy triggers state doctype](#cozy-triggers-state-doctype)
+  - [`io.cozy.triggers.state`](#iocozytriggersstate)
+
+# Cozy triggers state doctype
+
+## `io.cozy.triggers.state`
```
Comment (nono) [inline docs/io.cozy.triggers.state.md]: I think we should make it clear that this doctype is not persisted in CouchDB, its data is recomputed from jobs.

File: docs/io.cozy.settings.md [hunk 12]
```diff
@@ -0,0 +1,12 @@
+- [Cozy settings doctype](#cozy-settings-doctype)
+  - [`io.cozy.settings`](#iocozysettings)
+
+# Cozy settings doctype
+
+## `io.cozy.settings`
+
+The `io.cozy.settings` doctype contains all the stack instance-related settings.
+
```
Comment (gregorylegarec) [inline docs/io.cozy.settings.md]: I think `lang` is also an attribute of `io.cozy.settings`

**Pull request merged**